pkg/crypto/keccak/keccak.go

```diff
@@ -18,8 +18,10 @@ import (
 // [hash function]: https://docs.starknet.io/docs/Hashing/hash-functions#starknet-keccak
 func Digest250(data []byte) *big.Int {
 	digest := Digest256(data)
-	digest[0] >>= 6 // Clear the last 6 bits.
-	return new(big.Int).SetBytes(digest)
+	keccak_data := new(big.Int).SetBytes(digest)
+	MASK_250, _ := new(big.Int).SetString("3ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 16)
+
+	return keccak_data.And(keccak_data, MASK_250)
```

Comment:
Great work figuring out! A few improvements we could make is to remove the extra allocation and then adjust the naming to follow Go conventions as follows.

```go
digest := new(big.Int).SetBytes(Digest256(data))
mask, _ := new(big.Int).SetString("3ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 16)
return digest.And(digest, mask)
```